pkgs/applications/editors/rstudio/default.nix

```diff
@@ -73,24 +53,39 @@ let
     hash = "sha256-lZnZvioztbBWWa6H177X6rRrrgACx2gMjVFDgNup93g=";
   };
 
-  description = "Set of integrated tools for the R language";
 in
-(if server then stdenv.mkDerivation else mkDerivation)
-  (rec {
-    inherit pname version src RSTUDIO_VERSION_MAJOR RSTUDIO_VERSION_MINOR RSTUDIO_VERSION_PATCH RSTUDIO_VERSION_SUFFIX;
+stdenv.mkDerivation rec {
+  pname = "RStudio";
+  version = "2024.04.2+764";
+
+  RSTUDIO_VERSION_MAJOR = lib.versions.major version;
+  RSTUDIO_VERSION_MINOR = lib.versions.minor version;
+  RSTUDIO_VERSION_PATCH = lib.versions.patch version;
+  RSTUDIO_VERSION_SUFFIX = "+" + toString (lib.tail (lib.splitString "+" version));
+
+  src = fetchFromGitHub {
+    owner = "rstudio";
+    repo = "rstudio";
+    rev = "v" + version;
```

Comment:
```suggestion
    rev = "refs/tags/v${version}";
```

Reply:
Will do later, but wanted the first few commits to be NOOPs